scripts/playbooks.py

```diff
@@ -154,6 +154,45 @@ def is_in_collection(operation: DagOperation) -> bool:
             else:
                 all_fd.write(f"# {operation.name}\n")
 
+    with Path(meta_dir, "start.yml").open("w") as start_fd:
+        write_copyright_licence_headers(start_fd)
+        start_fd.write("---\n")
+        for operation in dag.get_all_operations(restart=False, stop=False):
+            if for_collection and not is_in_collection(operation):
+                continue
+            if not is_noop(operation) and "start" in operation.name.action:
+                start_fd.write(
+                    f"- ansible.builtin.import_playbook: {playbooks_prefix}{operation.name}.yml\n"
+                )
+            else:
+                start_fd.write(f"# {operation.name}\n")
```

Comment:
Can you prefix 2 `#` for noop operation and 1 `#` for "normal" operation. This way we can see what operation is noop.